internal/helm/flags/flag.go

```diff
@@ -119,6 +120,11 @@ func (f *Flags) AddTo(flagSet *pflag.FlagSet) {
 			" holding the leader lock (required if running locally with leader"+
 			" election enabled).",
 	)
+	flagSet.BoolVar(&f.SuppressOverrideValues,
+		"suppress-override-values",
+		false,
+		"Manage log events about overriding template values",
```

Comment:
Nit: This description is a bit vague IMO. Maybe something like "hides the override value for OverrideValuesInUse events"?

Reply:
Thank you - I improved it.